crates/cubecl-common/src/tma.rs

```diff
@@ -0,0 +1,96 @@
+use alloc::vec::Vec;
+use serde::{Deserialize, Serialize};
+
+/// Format of [`TensorMap`]
+#[derive(Hash, PartialEq, Eq, Clone, Debug, Serialize, Deserialize)]
+pub enum TensorMapFormat {
```

Comment:
Ho I don't think this should do in cubecl-common! cubecl-common is for stuff like stub mutex, stub hashmap, id generation, utilities that are quite general. It is reexported in burn-common as well. This file should probably go in `cubecl-core` or `cubecl-ir`

Reply:
The reason I put it there is because there's currently a separation between the compiler side with the IR and frontend, and the runtime. These types are used by both, so I put them in common.

Reply:
It's a question of whether `cubecl-runtime` should add core, or put this in common. I think it's more sensible to put it in common.

Reply:
I moved it to `cubecl-runtime` instead